content/rest-api/responses.md

```diff
@@ -214,6 +214,16 @@ HTTP/1.1 422 Unprocessable Entity
 }
 ```
 
+### 429 error
+Over solliciting the API results in a `429 Too Many Requests` response.
+A Retry-After header is set in the response to indicate the time to wait before the next retry.
+
+#### Example
+```http
+HTTP/1.1 429 Too Many Requests
+Retry-After: 15
+```
+
```

Comment:
I think we should move this part below the `tips` part which is linked to 422 ;)

Reply:
I saw the `tips` part and wondered if it was related to 422, but reading it, I thought it was more related to 404 and so at the bottom of all the client errors.

That's why I left it at the end.